src/net/sourceforge/kolmafia/persistence/CoinmastersDatabase.java

```diff
@@ -236,7 +244,8 @@ public static int purchaseLimit(final int itemId) {
       case ItemPool.ZEPPELIN_TICKET,
           ItemPool.TALES_OF_DREAD,
           ItemPool.BRASS_DREAD_FLASK,
-          ItemPool.SILVER_DREAD_FLASK -> 1;
+          ItemPool.SILVER_DREAD_FLASK,
+          ItemPool.MINI_KIWI_INTOXICATING_SPIRITS -> 1;
```

Comment:
brass dread flask is once / ever, intoxicating spirits is once / day, does this work correctly?

Reply:
right, the actual pref stuff is handled in the purchase request

Reply:
I think the point of this is generating a purchase request with a max of 1, so if you look at the CoinMasterPurchaseRequest in the MallSearchFrame, it won't tell you that you can buy "unlimited" of them.